option optimize_for = LITE_RUNTIME;

message VisitGetRewardRequest
{
 required uint32 route_id = 1;
}
